src/richie/apps/search/indexers/courses.py

```diff
@@ -130,6 +137,7 @@ def build_es_query(request, facets):
         # we pass them to ES. See the docs en bool queries.
         # https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html
         queries = []
+        key_fragment_pair = namedtuple("key_fragment_pair", ["key", "fragment"])
```

Comment:
In fact the namedtuple should be declared on top of the file so its instanciated at code evaluation once for all.

Reply:
But won't that require it to be in all caps like an export variable? Or does that not matter?

Reply:
IMO no problem. Hope pylint agrees !